src/main/java/com/wix/mysql/distribution/Version.java

```diff
@@ -1,94 +1,49 @@
 package com.wix.mysql.distribution;
 
-import static java.lang.String.format;
-
 import com.wix.mysql.exceptions.UnsupportedPlatformException;
 import com.wix.mysql.utils.Utils;
 import de.flapdoodle.embed.process.distribution.IVersion;
 import de.flapdoodle.embed.process.distribution.Platform;
+
 import java.util.Arrays;
 import java.util.List;
 
+import static java.lang.String.format;
+
 public enum Version implements IVersion {
 
-    v5_5_40("5.5", 40, MacOsVersion.v10_6, Platform.Linux, Platform.OS_X),
-    v5_5_41("5.5", 41, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_42("5.5", 42, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_43("5.5", 43, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_44("5.5", 44, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_45("5.5", 45, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_46("5.5", 46, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_47("5.5", 47, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_48("5.5", 48, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_49("5.5", 49, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_50("5.5", 50, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_51("5.5", 51, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_52("5.5", 52, MacOsVersion.v10_9, Platform.Linux, Platform.OS_X),
-    v5_5_latest(v5_5_52),
-    v5_6_21("5.6", 21, MacOsVersion.v10_9),
-    v5_6_22("5.6", 22, MacOsVersion.v10_9),
-    v5_6_23("5.6", 23, MacOsVersion.v10_9),
-    v5_6_24("5.6", 24, MacOsVersion.v10_9),
-    v5_6_25("5.6", 25, MacOsVersion.v10_9),
-    v5_6_26("5.6", 26, MacOsVersion.v10_9),
-    v5_6_27("5.6", 27, MacOsVersion.v10_9),
-    v5_6_28("5.6", 28, MacOsVersion.v10_10),
-    v5_6_29("5.6", 29, MacOsVersion.v10_10),
-    v5_6_30("5.6", 30, MacOsVersion.v10_11),
-    v5_6_31("5.6", 31, MacOsVersion.v10_11),
-    v5_6_32("5.6", 32, MacOsVersion.v10_11),
-    v5_6_33("5.6", 33, MacOsVersion.v10_11),
-    v5_6_latest(v5_6_33),
-    v5_7_10("5.7", 10, MacOsVersion.v10_10),
-    v5_7_11("5.7", 11, MacOsVersion.v10_10),
-    v5_7_12("5.7", 12, MacOsVersion.v10_11),
-    v5_7_13("5.7", 13, MacOsVersion.v10_11),
-    v5_7_14("5.7", 14, MacOsVersion.v10_11),
-    v5_7_15("5.7", 15, MacOsVersion.v10_11),
-    v5_7_latest(v5_7_15);
-
-    private enum MacOsVersion {
-        v10_6, v10_7, v10_8, v10_9, v10_10, v10_11;
-
-        @Override
-        public String toString() {
-            return name().substring(1).replace('_', '.');
-        }
-    }
+    v5_5_40("5.5", "40", Platform.Linux, Platform.OS_X),
+    v5_6_21("5.6", "21"),
+    v5_6_22("5.6", "22"),
+    v5_6_23("5.6", "23"),
+    v5_6_24("5.6", "24"),
+    v5_6_latest(v5_6_24),
+    v5_7_10("5.7", "10"),
+    v5_7_13("5.7", "10"),
```

Comment:
Interestingly enough, the build consistently hangs on at least two Sierra laptops I tried it on. I guess the MySQL bug is only reproducible under certain conditions, but no idea what they might be.